src/network/network_gui.cpp

```diff
@@ -448,7 +462,7 @@ class NetworkGameWindow : public Window {
 	}
 
 public:
-	NetworkGameWindow(WindowDesc *desc) : Window(desc), name_editbox(NETWORK_CLIENT_NAME_LENGTH), filter_editbox(120)
+	NetworkGameWindow(WindowDesc *desc) : Window(desc), name_editbox(NETWORK_CLIENT_NAME_LENGTH), filter_editbox(120), searched_internet(false)
```

Comment:
Wasn't the consensus to use `bool searched_internet = false; ///< Did we ever...` instead?